wandb/sdk/backend/backend.py

```diff
@@ -65,7 +64,7 @@ def __init__(
         mailbox: Mailbox,
         settings: Optional[Settings] = None,
         log_level: Optional[int] = None,
-        manager: Optional[_Manager] = None,
+        service: "Optional[service_connection.ServiceConnection]" = None,
```

Comment:
i think it is enough to put the `service_connection.ServiceConnection` in quotes